languages/elixir/exercises/concept/basic-strings/.meta/design.md

```diff
@@ -0,0 +1,51 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student how strings are implemented and used in Elixir.
+
+## Learning objectives
+
+After completing this exercise, the student should:
+
+- Know how to concatenate strings.
+- Know how to manipulate strings.
+- Know how to perform string interpolation.
+- Know how to create multiline strings (heredocs).
```

Comment:
I couldn't find any concrete sources about this in Elixir so that I could link for the students or even properly educate myself. The only reference to `"""` that actually names it instead of just silently using, is in the [Getting Started guide about sigils](https://elixir-lang.org/getting-started/sigils.html):

> Sigils also support heredocs, that is, triple double- or single-quotes as separators

But I don't want to mention sigils in this exercise, they seem too advanced to me.

Reply:
https://elixir-examples.github.io/examples/multiline-strings-heredocs has been a good resource for me

Reply:
Thanks! I linked it in hints.md and after.md.